packages/blocks/src/api/raw-handling/special-comment-converter.js

```diff
@@ -105,7 +67,35 @@ function moreCommentConverter( node, doc ) {
 	}
 }
 
-function createMore( customText, noTeaser, doc ) {
+function createBlock( commentNode, doc ) {
```

Comment:
I understand the convenience of folding everything into a single function. However, the resulting function is more difficult to understand for someone who doesn't have our context. Can we add a comment explaining that this function handles both `more` and `nextpage` comments, for ease of maintenance?

If we wanted to go the extra mile, we could even make that explicit in code, either with a switch:

```js
switch ( commentNode.nodeValue.match(/\w+/)?.[0] ) {
  case 'nextpage':
    // do stuff
    break;
  case 'more':
    // do stuff
    break;
}
```

Or simply via function delegation:

```js
function createBlock( commentNode, doc ) {
  if ( commentNode.nodeValue === 'nextpage' ) {
    return createNextPage( doc );
  }
  return createMore( commentNode, doc );
}
```

Reply:
Restore the original functions :)